packages/buckets/src/buckets.spec.ts

```diff
@@ -210,27 +215,41 @@ describe('Buckets...', function () {
     })
 
     it('should pull files by path and write to file on node', async function () {
-      if (isBrowser) return this.skip()
       // Bucket path
       const rootKey = buck.root?.key || ''
       let length = 0
-
       // Bucket path
       const chunks = client.pullPath(rootKey, 'dir1/file1.jpg', {
         progress: (num) => (length = num || 0),
       })
-      const pth = path.join(__dirname, '../../..', 'testdata')
-      const stream = fs.createWriteStream(path.join(pth, 'output.jpg'))
-      // Should be an AsyncIterable
-      for await (const chunk of chunks) {
-        stream.write(chunk)
+      if (isBrowser) {
+        let result = new Uint8Array();
+        const append = (target: Uint8Array, addition: Uint8Array) => {
```

Comment:
If we don't have this in a util module somewhere, we probably should. I'd guess we use this pattern elsewhere.

Reply:
with all the packages moving together, it seems like a nice enhancement in the near future will be:

1. a unified set of utils that users might want across the board
2. a unified set of utils used in tests all over the place